pkg/services/accesscontrol/evaluator/evaluator.go

```diff
@@ -36,21 +40,28 @@ func evaluateScope(dbScopes map[string]struct{}, targetScopes ...string) (bool,
 		return true, nil
 	}
 
+	logger := log.New("accesscontrol.permissioncheck")
 	for _, s := range targetScopes {
+
 		var match bool
 		for dbScope := range dbScopes {
+			if strings.ContainsAny(s, "*?") {
+				logger.Error(fmt.Sprintf("Invalid scope: %v should not contain meta-characters like * or ?", s, dbScope))
```

Comment:
Should we not return an error here?

Reply:
I thought logger.Error panicked. Apparently not.